### PR TITLE
Update the VS Code API and use an output window to log messages

### PR DESCRIPTION
**Pull request by tonybaloney**

### CODE REVIEW

File: src/extension.ts [hunk 1]
```diff
@@ -13,8 +13,10 @@ export function activate(context: vscode.ExtensionContext) {
 		clearDecorations();
 	});
 
+	const output :vscode.OutputChannel = vscode.window.createOutputChannel("Austin");
```
Comment (P403n1x87) [inline src/extension.ts:16]: Can we create this inside the `AustinController` so that we don't have to pass it in via the constructor?

**Comment by tonybaloney** [comment]
Updates made

File: src/controller.ts [hunk 2]
```diff
@@ -10,8 +10,9 @@ function delay(ms: number) {
 
 
 export class AustinController {
+    private output: vscode.OutputChannel = vscode.window.createOutputChannel("Austin");
 
-    public constructor(private stats: AustinStats) { }
+    public constructor(private stats: AustinStats, ) { }
```
Comment (P403n1x87) [inline src/controller.ts:15]:
```suggestion
    public constructor(private stats: AustinStats) { }
```

**Review by P403n1x87 — CHANGES_REQUESTED** [review]
Great! Just a nit otherwise LGTM.

**Review by P403n1x87 — APPROVED** [review]

**Pull request merged**